server/routes/conversations.js

```diff
@@ -0,0 +1,51 @@
+var express = require('express');
+var passport = require("passport");
+var router = express.Router();
+var conversationSchema = require('../models/conversationSchema');
+require('mongoose').Promise = global.Promise
+
+router.get('/', async (req, res, next) => {
+    res.status(200).send({
+        message: "conversations page!"
+    })
+})
+
+router.post('/addConversation/', passport.authenticate('jwt', {session: false}), async (req, res, next) => {
```

Comment:
nit: this route path can be `/add` since it's prefixed by `/conversation/` by router